modules/ROOT/pages/vantagecloud-lake-demo-jupyter-docker.adoc

```diff
@@ -0,0 +1,126 @@
+
+== Overview
+In this how-to we will go through the steps for connecting to TeradataCloud Lake and run demos from a Jupyter notebook in Docker. 
+
+== Prerequisites
+* https://www.docker.com/products/docker-desktop/[Docker Desktop] installed
+* (Optional) https://git-scm.com/book/en/v2/Getting-Started-Installing-Git[Git] installed
+** Required to download git repo from https://github.com/Teradata/lake-demos.git. You can also download the repo as a .zip if you don't have Git installed locally.
+* A Teradata VantageCloud Lake account login
+** Organization URL and login details from Teradata welcome letter
+* (Optional) IDE of your choice
+
+== Create VantageCloud Lake environment
+Follow the instructions from the https://quickstarts.teradata.com/getting-started-with-vantagecloud-lake.html[VantageCloud Lake getting started] to create your own environment. +
+Once created, go to *SETTINGS* tab and provide your IP address range in CIDR format (for example, 192.168.2.0/24 specifies all IP addresses in the range: 192.168.2.0 to 192.168.2.255) to access the environment.
+
+image::lake_ip_addresses.png[IP whitelisting,align="center",width=50%]
+
+Your environment card should show *Public internet* access now.
+
+image::lake_public_internet_cv.png[Public internet card view,align="center",width=50%]
+
+From *OVERVIEW* tab, copy:
+
+* Public IP and
+* Open Analytics Endpoint
+
+These values are required to access VantageCloud Lake from outside.
+
+image::lake_overview_page.png[Environment Overview page,align="center",width=75%]
+
+== Clone Lake demo repository
+Clone VantageCloud Lake Demo repository in your local machine:
+
+[source, bash, id="jupyter_first_run", role="content-editable emits-gtm-events"]
+----
+git clone https://github.com/Teradata/lake-demos.git
+----
+
+The repository contains different files and folders, the important ones are:
+
+* Jupyter Notebooks
+** https://github.com/Teradata/lake-demos/blob/main/0_Demo_Environment_Setup.ipynb[0_Demo_Environment_Setup.ipynb]
+** https://github.com/Teradata/lake-demos/blob/main/1_Load_Base_Demo_Data.ipynb[1_Load_Base_Demo_Data.ipynb]
+** https://github.com/Teradata/lake-demos/blob/main/Data_Engineering_Exploration.ipynb[Data_Engineering_Exploration.ipynb]
+** https://github.com/Teradata/lake-demos/blob/main/Data_Science_OAF.ipynb[Data_Science_OAF.ipynb]
+** https://github.com/Teradata/lake-demos/blob/main/Demo_Admin.ipynb[Demo_Admin.ipynb]
+* https://github.com/Teradata/lake-demos/blob/main/vars.json[vars.json file]
+
+== Edit vars.json file
+To connect Jupyter notebooks with VantageCloud Lake, you need to edit "vars.json" file and provide:
+
+[cols="1,1"]
+|====
+| *Variable* | *Value*
+
+| *"host"* 
+| Public IP value from *OVERVIEW* section (see above)
+
+| *"UES_URI"* 
+| Open Analytics Endpoint value from *OVERVIEW* section (see above)
+
+| *"bucket"* 
+| S3 bucket name +
+  *If you do not have it, reach out to Teradata team.
```

Comment:
There is an email thread where we are taking about moving this data to a public bucket. Let's discuss in the standup today. how to sequence this.